app/views/templates/playlists.jst

```diff
@@ -0,0 +1,15 @@
+<h4><%= t('your music') %></h4>
+<p id="all-song" class="selected"><%= t('all songs') %></p>
+<p id="up-next"><%= t('up next') %></p>
```

Comment:
This two are probably wrapped in a `ul` elements and converted to `li` (as they are items in a list)